src/foam/nanos/crunch/rules.jrl

```diff
@@ -21,8 +21,9 @@ p({
   "documentation":"Re-assign user spid on CREATE of User-ServiceProvider junction",
   "daoKey":"userCapabilityJunctionDAO",
   "operation":0,
-  "after":true,
+  "after":false,
   "enabled":true,
+  "predicate":{ "class":"foam.nanos.crunch.predicate.IsUserServiceProviderJunction" },
   "action":{
     "class":"foam.nanos.auth.SetUserServiceProviderJunctionRuleAction"
   },
```

Comment:
Do we really need this rule?  I don't see a scenario where we'll be "switching" a user's spid by adding a user capability junction. Most cases we set a new spid value on an object. We have another rule which checks for this spid update on user and handles the setup of the spid capabilities.

Reply:
i think thats fine if we don't need to cover this scenario, but then we might want prevent puts to "userCapabilityJunctionDAO" for user-serviceprovider junctions altogether